terraform/application/statuscake.tf

```diff
@@ -0,0 +1,9 @@
+module "statuscake" {
+  for_each = var.statuscake_alerts
```

Comment:
We don't need a for_each if there is only 1 entry in the map

Reply:
done